theme.json

```diff
@@ -44,6 +44,11 @@
 					"color": "#503AA8",
 					"name": "Accent 3",
 					"slug": "accent-3"
+				},
+				{
+					"color": "#F6CFF4",
```

Comment:
We don't need one as we already have it set with the "Accent 2". Please remove it.